config/standard/contact-summary.templated.js

```diff
@@ -191,11 +205,11 @@ cards = [
     label: 'contact.profile.immunizations',
     appliesToType: 'person',
     appliesIf: function() {
-      return context.use_cases.imm && getAgeInMonths() < 144;
+      return context.use_cases.imm && extras.getAgeInMonths() < 144;
```

Comment:
passing in the contact would have made this 'more explicit' - (you probably found it as is)

Reply:
sorry - didn't understand this one :(

Reply:
`extras.getAgeInMonths(contact)` VS `extras.getAgeInMonths()`